### ISSUE TITLE
Multipage form submission error

### ISSUE THREAD

**Issue by samroberts707** [verbb/formie#1454]
### Describe the bug

When pressing submit no error messages appear except a console error:

```
Uncaught TypeError: Cannot read properties of null (reading 'textContent')
    at HTMLButtonElement.eval (formie-form-theme.js:187:59)
```

I can see the submission in the formie file logs but nothing show in either the Admin or Frontend. 
I've completely removed all caching on the site to see if that would help too.

### Steps to reproduce

1. 5 page multi-page form, first page has 4 text fields and then remaining pages have radio and checkbox fields.
2. I'm rendering the form myself and placing each page inside a Flickity carousel slide. 

### Form settings

- Multi-page form: Yes
- Submission Method: Ajax or Page Reload - I've tried both and getting the same error
- Client-side Validation: Yes 
- Custom Form Templates: Yes

### Craft CMS version

4.4.13

### Plugin version

2.0.30

### Multi-site?

No

### Additional context

Happy to provide template twig code if that would be useful in diagnosing issue.

**Comment by engram-design** [comment]
Updated for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.0.30"`.

**Comment by engram-design** [comment]
Fixed in [2.0.31](https://github.com/verbb/formie/releases/tag/2.0.31)
